### PR TITLE
fix: single tech sample folder structure for synapse and adf

### PR DESCRIPTION
**Pull request by devlace**
# Type of PR

- Documentation changes
- Code changes

## Purpose

- This moves files in the single tech sample into the correct structure.
- Move existing adf single tech sample into this folder: `single_tech_samples/datafactory/sample1_cicd`
- Move devcontainer in synapse sample into sample1

## Author pre-publish checklist

- [ ] Added test to prove my fix is effective or new feature works
- [X] No PII in logs
- [X] Made corresponding changes to the documentation

### CODE REVIEW

File: single_tech_samples/datafactory/README.md [hunk 1]
```diff
@@ -1,282 +1,6 @@
-# DataOps with Azure Data Factory <!-- omit in toc -->
+# Azure Synapse Analytics
```
Comment (promisinganuj) [inline single_tech_samples/datafactory/README.md:1]: This should be "Data Factory"

File: single_tech_samples/datafactory/sample1_cicd/README.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+       - **RESOURCE_GROUP_LOCATION** - Azure location to deploy resources. *Default*: `westus`.
+       - **AZURE_SUBSCRIPTION_ID** - Azure subscription id to use to deploy resources. *Default*: default azure subscription. To see your default, run `az account list`.
+       - **RESOURCE_GROUP_NAME_PREFIX** - name of the resource group. This will automatically be appended with the environment name. For example: `RESOURCE_GROUP_NAME_PREFIX-dev-rg`. *Default*: mdwdo-ado-${DEPLOYMENT_ID}.
+       - **DEPLOYMENT_ID** - string appended to all resource names. This is to ensure uniqueness of azure resource names. *Default*: random five character string.
+       - **AZDO_PIPELINES_BRANCH_NAME** - git branch where Azure DevOps pipelines definitions are retrieved from. *Default*: main.
+
+      To further customize the solution, set parameters in `arm.parameters` files located in the `infrastructure` folder.
+
+2. **Deploy Azure resources**
+   1. `cd` into the `single_tech_samples/sample1_cicd/datafactory` folder of the repo
```
Comment (promisinganuj) [inline single_tech_samples/datafactory/sample1_cicd/README.md:198]: This should be "single_tech_samples/datafactory/sample1_cicd"

**Review by promisinganuj — CHANGES_REQUESTED** [review]
Two minor comments otherwise LGTM!

File: single_tech_samples/datafactory/sample1_cicd/README.md (hunk 2 above)
Comment (devlace) [reply single_tech_samples/datafactory/sample1_cicd/README.md:198]: Good catch! Wrong cursor in copy paste process 🙃

File: single_tech_samples/datafactory/README.md (hunk 1 above)
Comment (devlace) [reply single_tech_samples/datafactory/README.md]: Ah another copy paste error :) Fixed.

File: single_tech_samples/datafactory/sample1_cicd/README.md (hunk 2 above)
Comment (devlace) [reply single_tech_samples/datafactory/sample1_cicd/README.md]: Fixed

**Review by promisinganuj — APPROVED** [review]
lgtm!

**Pull request merged**